phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php"
+		 colors="true">
+
+	<testsuite name="Default">
+		<directory>tests/unit/</directory>
+	</testsuite>
+	<filter>
+		<whitelist processUncoveredFilesFromWhitelist="true">
+			<directory suffix=".php">src/</directory>
+		</whitelist>
+	</filter>
+
+	<groups>
+		<exclude>
+			<group>sanitychecks</group>
+		</exclude>
+	</groups>
+</phpunit>
```

Comment:
Can we use spaces for indentation please? Might need an editorconfig rule bump to go with